README.md

```diff
@@ -229,9 +232,10 @@ Services:
  - [ProvenDB](https://www.provendb.com/) -  Blockchain based Data integrity solution for MongoDB
 
 ### Deployment
+ - [Cluster to cluster sync](https://www.mongodb.com/products/cluster-to-cluster-sync) - Continuous data sync between separate MongoDB clusters
```

Comment:
Oh, that truly is awesome! Can you please put this under a `Services:` section to stay consistent and explicit?